src/io/PipeWriter.zig

```diff
@@ -1246,15 +1320,15 @@ pub fn WindowsStreamingWriter(
         }
 
         pub fn writeUTF16(this: *WindowsWriter, buf: []const u16) WriteResult {
-            return writeInternal(this, buf, StreamBuffer.writeUTF16);
+            return writeInternal(this, buf, &StreamBuffer.writeUTF16);
```

Comment:
with this being so tiny is this a candidate for:
`pub inline fn writeUTF16`?

(same with the ones below)